docs/api-table.js

```diff
@@ -779,11 +779,11 @@ export const react = {
       ['noVisibleLabel', 'boolean', '', 'Whether label should be invisible'],
       ['checked', 'boolean', '', 'Whether the single option should be checked (controlled)'],
       ['defaultChecked', 'boolean', '', 'Whether the single option should be checked on mount (uncontrolled)'],
-      ['small', 'boolean', '', 'Whether the elements should be small'],
-      ['optional', 'boolean', '', 'Whether the toggle is optional Appends (valgfritt) to the end of the title for indication'],
-      ['equalWidth', 'boolean', '', 'Will make each radio-button equal width'],
       ['indeterminate', 'boolean', '', 'Whether a single option is indeterminate, or "partially checked." The checkbox will appear with a small dash instead of a tick to indicate that the option is not exactly checked or unchecked.'],
+      ['optional', 'boolean', '', 'Whether the toggle is optional Appends (valgfritt) to the end of the title for indication'],
```

Comment:
Should the Norwegian word `valgfritt` be here? :)

Reply:
Oh I just realized that you had just moved this line and that it wasn't your changes, so maybe we should save my previous question for another ticket? 😄

Reply:
Still, good point - missed that :D Fixed in [0854c34](https://github.com/warp-ds/tech-docs/pull/189/commits/0854c3419d9769f2e8702ab51d3f761924d64e06)